CHANGELOG.md

```diff
@@ -15,8 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Ensure that availability zones are kept unchanged during migration from 12.x to 13.x.
 - Don't set `MachinePool.Status.InfrastructureReady` in `nodestatus` handler.
-- Update `giantswarm/conditions-handler` to `v0.1.2`.
```

Comment:
Why did you remove this?

Reply:
There is this few lines above:

```
- Bump `conditions-handler` to v0.2.0 to get `MachinePool` `ReplicasReady` condition.
```
I updated PR description.

Reply:
Forgot to update it before :)